policies/github/organization.rego

```diff
@@ -110,3 +110,20 @@ default organization_not_using_single_sign_on = true
 organization_not_using_single_sign_on = false {
 	input.saml_enabled
 }
+
+# METADATA
+# scope: rule
+# custom:
+#   severity: MEDIUM
+# title: Organization's Enterprise Should Disable Visibility Change Policy
+# description: It is recommended to change the enterprise's Repository visibility change policy. This will prevents users from creating private repositories and change them to be public. Malicous actors could leak code if enabled.
+# custom:
+#   remediationSteps: [Make sure you are an enterprise owner, Go to the policies page, Under the "Repository visibility change" section, choose the "Disabled" option]
+#   requiredScopes: [admin:enterprise]
+#   threat:
+#     - "A member of the organization could inadvertently or maliciously make public an internal repository exposing confidential data."
+default enterprise_not_using_visibility_change_disable_policy = true
```

Comment:
need to add prerequisite to test whether 'enterprise' was provided otherwise it will show the policy as "PASSES" instead of "SKIPPED"